package.json

```diff
@@ -97,10 +98,9 @@
   "dependencies": {
     "abort-controller": "^3.0.0",
     "any-signal": "^2.1.2",
-    "bignumber.js": "^9.0.0",
     "cids": "^1.1.6",
     "debug": "^4.2.0",
-    "ipfs-core-types": "^0.3.1",
+    "ipfs-core-types": "next",
```

Comment:
we need to get rid of these circular dependencies

Reply:
I don't think this is even required now, since we can pull `Blockstore` from `ipfs-repo` it's only in the tests, and even then it was only being used to pull types for the repo, which are in `ipfs-repo` now so it can go.